src/main/scala/com/evolutiongaming/crypto/DecryptConfig.scala

```diff
@@ -16,6 +19,9 @@ object DecryptConfig {
     } else {
       password
     }
+  } catch {
+    case th: DecryptAuthException => throw th
+    case NonFatal(_) => password
```

Comment:
hm… can we be more specific here?

Reply:
What do you mean?

Reply:
I mean that we'd like to fallback to password only in case of some specific errors and not all `NonFatal`, unless I'm missing something

Reply:
I want to return the same behaviour that we have in 1.5.1. I'm afraid there can be various kinds of exceptions thrown (now or in a future) and the common and most natural denominator is `NonFatal` 🤔